app/src/main/res/layout/activity_help.xml

```diff
@@ -55,9 +59,10 @@
         android:id="@+id/nav_view"
         android:layout_width="wrap_content"
         android:layout_height="match_parent"
-        android:layout_gravity="start"
+        android:layout_gravity="right"
```

Comment:
Why was this changed? "start" is correct here. It will automatically adjust if a rtl language is active as the start would then be on the right side.